vault/overlays/nerc-ocp-infra/backup-job/externalsecret.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: vault-backup-s3-endpoint
+spec:
+  secretStoreRef:
+    name: nerc-cluster-secrets
+    kind: ClusterSecretStore
+  refreshInterval: "1h"
+  target:
+    name: vault-backup-s3-endpoint
+    creationPolicy: 'Owner'
+    deletionPolicy: "Retain"
+    template:
+      engineVersion: v2
+  data:
+  - secretKey: ACCESS_KEY_ID
+    remoteRef:
+      key: $ENV/$CLUSTER/vault/backup/s3
```

Comment:
Are these paths correct? Where do `$ENV` and `$CLUSTER` come from?

Reply:
Good catch, updated